externs/lcevc.js

```diff
@@ -16,58 +16,47 @@ var LcevcDil = {};
 LcevcDil.LcevcDIL = class {
   /**
      *  @param {HTMLVideoElement} media
```

Comment:
formatting: The `*` characters in this comment are misaligned with the start of the comment.  They should look like this:

```js
/**
 * @param ...
 */
```